lib/core/wireprotocol/query.js

```diff
@@ -37,9 +37,13 @@ function query(server, ns, cmd, cursorState, options, callback) {
 
   // If we have explain, we need to rewrite the find command
   // to wrap it in the explain command
-  const explain = Explain.fromOptions(options);
-  if (explain) {
-    findCmd = decorateWithExplain(findCmd, explain);
+  try {
+    const explain = Explain.fromOptions(options);
```

Comment:
NODE-3377
Totally expanding scope here but we're validating too strictly here, this could be a quick fix that goes in this PR, we just need to assert string or boolean.